Anytype/Sources/PresentationLayer/Flows/SpaceHub/SpaceHubCoordinatorViewModel.swift

```diff
@@ -119,34 +119,41 @@ final class SpaceHubCoordinatorViewModel: ObservableObject {
         await spaceSetupManager.registerSpaceSetter(sceneId: sceneId, setter: activeSpaceManager)
         await setupInitialScreen()
         await handleVersionAlerts()
+        
+        startHandleWorkspaceInfo()
```

Comment:
You can't guarantee task order.

Reply:
You can use task group for start two infinity task after setup and guarantee there cancellation